src/mscorlib/src/System/Environment.cs

```diff
@@ -241,7 +241,7 @@ public static String MachineName
                 if (AppDomain.IsAppXModel() && !AppDomain.IsAppXDesignMode())
                 {
                     // Getting Computer Name is not a supported scenario on Store apps.
```

Comment:
The MachineName property should be deleted in CoreLib. The actual implementation was moved to corefx (https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Extensions/src/System/Environment.Uap.cs#L27).